### ISSUE TITLE
Dragging stacks of blocks over other stacks disconnects them

### ISSUE THREAD

**Issue by griffpatch** [LLK/scratch-blocks#922]
### Expected Behavior

Dragging a stack of blocks around the screen has no lasting impact on other blocks / scripts that are not part of the drag source or target.

### Actual Behavior

As a stack of two or more blocks is dragged over other block stacks, the other stacks become disconnected (although they stay positioned as if they were still connected).

### Steps to Reproduce

Here's a screen capture of me showing the behavior of dragging two blocks over a stack in a chrome browser. Notice how 'all' blocks are detached once the drag is complete even though I didn't release the drag over the stack? This is happening on https://llk.github.io/scratch-gui/ has this page got the latest changes from github?

![6cb209ba-4792-11e7-8535-7db61f9e0f1c](https://cloud.githubusercontent.com/assets/6737342/26799564/0aca5e84-4a2e-11e7-84f0-c622f66ed229.gif)

### Operating System and Browser

Windows 10 with Chrome

**Comment by rachel-fenichel** [comment]
This line is probably the culprit: https://github.com/LLK/scratch-blocks/blob/develop/core/connection.js#L376
